cmd/blockchaincmd/upgradecmd/apply.go

```diff
@@ -254,7 +254,6 @@ func applyPublicNetworkUpgrade(blockchainName, networkKey string, sc *models.Sid
 	if print {
 		blockchainIDstr := "<your-blockchain-id>"
 		if sc.Networks != nil &&
```

Comment:
should we check for ` if _,ok:=sc.Networks[networkKey]; !ok`

Reply:
for existance for the key in map so `sc.Networks[networkKey].BlockchainID` doesn't crash

Reply:
no need to check. the map does not return a pointer, but a NetworkData struct. When the key does not exist,
it returns a struct with all values empty. so no crash when accessing BlockchainID (crash will happen if the map returns a pointer, and returns nil for unknown key, and you want to access BlockchainID over a nil pointer)